src/cascadia/TerminalApp/TerminalTab.cpp

```diff
@@ -25,23 +25,16 @@ namespace winrt
 
 namespace winrt::TerminalApp::implementation
 {
-    TerminalTab::TerminalTab(const Profile& profile, const TermControl& control)
-    {
-        _rootPane = std::make_shared<Pane>(profile, control, true);
-
-        _rootPane->Id(_nextPaneId);
-        _activePane = _rootPane;
-        _mruPanes.insert(_mruPanes.begin(), _nextPaneId);
-        ++_nextPaneId;
-
-        _Setup();
-    }
-
     TerminalTab::TerminalTab(std::shared_ptr<Pane> rootPane)
     {
         _rootPane = rootPane;
         _activePane = nullptr;
 
+        if (_rootPane->_IsLeaf())

```

Comment:
Below I tried to do this with the `activePane == nullptr` block below, but clearly that doesn't work. I think that block should be edited so that you do
```
_rootPane->FindPane(firstId)->SetActive();
```
because I didn't realize the FocusPane wouldn't do anything when I wrote that code. This would make sure something is always focused if you try to create from multiple panes

Reply:
Makes sense, thanks